examples/camera-app/linux/include/camera-device.h

```diff
@@ -40,7 +40,7 @@
 static constexpr uint32_t kMaxContentBufferSizeBytes = 4096;
 static constexpr uint32_t kMaxNetworkBandwidthMbps   = 128;
 static constexpr uint8_t kMaxConcurrentEncoders      = 1;
-static constexpr uint32_t kMaxEncodedPixelRate       = 27648000; // 720p at 30fps
```

Comment:
These constants would be clearer  as ` = 1280 * 720 * 30` so that we clearly see how we can compute these.

Reply:
Added the product of subparts